dashboard/assets/packages/ui/src/components/Stat/Stat.tsx

```diff
@@ -59,12 +67,8 @@ export default function Stat({ panel }) {
         {panel.title}
       </Typography>
       <div ref={ref}>
-        <UplotReact options={options} data={plot.data} />
+        <UplotReact options={options} data={plot.data as AlignedData} />
```

Comment:
Perhaps we could find a way for SeriesPlot to return the correct type? Type '(number | undefined)[][]' is not assignable to type 'AlignedData'.